lucene/lucene-directory/src/test/java/org/infinispan/lucene/locking/LockManagerFunctionalTest.java

```diff
@@ -33,24 +53,21 @@ protected TransactionMode getTransactionsMode() {
 
    @Test(dataProvider = "writeLockNameProvider")
    public void testLuceneIndexLocking(final String writeLockProvider) throws IOException {
-      final String commonIndexName = "myIndex";
-      LockFactory lockManagerA = makeLockFactory(cache(0,"lucene"), commonIndexName);
-      LockFactory lockManagerB = makeLockFactory(cache(1, "lucene"), commonIndexName);
-      LockFactory isolatedLockManager = makeLockFactory(cache(0, "lucene"), "anotherIndex");
-      Lock luceneLockA = lockManagerA.makeLock(writeLockProvider);
-      Lock luceneLockB = lockManagerB.makeLock(writeLockProvider);
-      Lock anotherLock = isolatedLockManager.makeLock(writeLockProvider);
+      BaseLockFactory baseLockFactory = BaseLockFactory.INSTANCE;
+      Lock luceneLockA = baseLockFactory.makeLock(directory1, writeLockProvider);
+      Lock luceneLockB = baseLockFactory.makeLock(directory1, writeLockProvider);
+      Lock anotherLock = baseLockFactory.makeLock(directory3, writeLockProvider);
 
       assert luceneLockA.obtain();
       assert luceneLockB.isLocked();
-      assert ! anotherLock.isLocked();
-      assert ! luceneLockA.obtain();
-      assert ! luceneLockB.obtain();
+      assert !anotherLock.isLocked();
```

Comment:
Maybe we could use standard `assertTrue` and `assertFalse` here? It would look consistent.